packages/sveltekit-plugin/src/sveltekit-bundle-analysis/sveltekitBundleAnalysisPlugin.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  type BundleAnalysisUploadPlugin,
+  red,
+} from "@codecov/bundler-plugin-core";
+import { getBundleName } from "./getBundleName";
+
+// @ts-expect-error this value is being replaced by rollup
+const PLUGIN_NAME = __PACKAGE_NAME__ as string;
```

Comment:
Let's define

```
declare global {
    __PACKAGE_NAME__: string;
   __PACKAGE_VERSION__: string;
}
```

in a TS declaration file somewhere so we can remove these. Feel free to do it in a follow up PR.